src/Drupal/Commands/core/EntityCommands.php

```diff
@@ -40,18 +44,22 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager)
      *   Delete all nodes except node 9, 14 and 81.
      * @usage drush entity:delete user
      *   Delete all users except uid=1.
+     * @usage drush entity:delete node --chunks=5
+     *   Delete all node entities in steps of 5.
      *
      * @command entity:delete
      * @aliases edel,entity-delete
      * @throws \Exception
      */
-    public function delete($entity_type, $ids = null, $options = ['bundle' => self::REQ, 'exclude' => self::REQ])
+    public function delete(OutputInterface $output, $entity_type, $ids = null, $options = ['bundle' => self::REQ, 'exclude' => self::REQ, 'chunks' => self::REQ])
```

Comment:
I didn't know that OutputInterface would work here. Look kosher to you, @greg-1-anderson? 


Also, lets just default chunks to 50 or so. We get advantages to bulk deleting so I think 10 is too few.

Reply:
I altered the default to 50 and fixed some coding standard issues. Why do you think the OutputInterface wouldn't work? I tested on my Drupal installation and it worked fine.